specification.md

```diff
@@ -196,6 +211,28 @@ Represents person registration data.
 }
 ```
 
+### RegistrationInfo
```

Comment:
Would you be open to including competitorLimit here as well? I'd like to use this, along with registrationOpen and registrationClose, for Cube New England's website, so I can say "87 / 125 spots filled".